src/main/java/com/homework/springboot/service/SchedulerService.java

```diff
@@ -0,0 +1,45 @@
+package com.homework.springboot.service;
+
+import com.homework.springboot.model.User;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@Service
+@Slf4j
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class SchedulerService {
+
+    private static final String CRON = "*/10 * * * * *";
```

Comment:
Please use a property file for cron.
https://stackoverflow.com/questions/24306004/task-scheduling-using-cron-expression-from-properties-file